net-im/bitlbee-facebook/bitlbee-facebook-9999.ebuild

```diff
@@ -8,18 +8,17 @@ inherit autotools
 DESCRIPTION="Facebook protocol plugin for BitlBee"
 HOMEPAGE="https://github.com/bitlbee/bitlbee-facebook"
 
-LICENSE="GPL-2"
-SLOT="0"
-IUSE=""
-
 if [[ ${PV} == *9999 ]] ; then
 	inherit	git-r3
 	EGIT_REPO_URI="https://github.com/bitlbee/bitlbee-facebook"
 else
 	SRC_URI="https://github.com/bitlbee/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
-	KEYWORDS="~amd64 ~x86"
+	KEYWORDS="amd64 x86"
```

Comment:
What are you doing here?

Reply:
It is an error indeed.

If 9999 exists, then do assume it is a template for future version bumps. New version bumps are not added as stable.